.ci/ci_cache_components.sh

```diff
@@ -189,6 +201,9 @@ main() {
 		r)
 			build_image_initrd="true"
 			;;
+		d)
+			built_with_dracut="true"
```

Comment:
Just an observation: the script now has a mix of variables with two very similar prefixes: `build_` and `built_`. That could easily lead to mistakes in the future if someone specifies `built` rather than `build` (or *vice versa*) :smile:

To guard against that, I'd consider adding `set -o nounset` to the top of the script. That would require you to change lines like `local built_with_dracut="${1}"` to `local built_with_dracut="${1:-}"`, but may be worth it to make the script as robust as possible.

Reply:
I actually struggled with the naming a bit.  On one hand, this script doesn't really build anything, it just deals with artifacts that have been built already so 'build' seems a bit misleading in that respect.  On the other hand, you're right that my naming is inconsistent with naming of similar variables.  Would going with 'build_with_dracut' be more acceptable?

Reply:
Thanks @pmores - sounds good to me.

Reply:
As for the latter point, I agree but I'd rather leave this to a separate PR.